tools/sanity_checks.py

```diff
@@ -77,6 +77,9 @@
     'openal-soft': [
         'hexify.py'
     ],
+    'physfs': [
+        'simple_test.c'
+    ],
```

Comment:
In general this is considered a bad idea, and the reason this override table exists is really for the sake of buildsystem helper scripts and not for source code.

Please try submitting this upstream, IMHO it would make sense for them to be willing to accept the contribution of an automated sanity-check test program.

Reply:
I've removed this. I'm not sure if it'll make it upstream any time soon, given that 3.2 branch is a bugfix-only and physfs 4 is still in the planning phase.

Reply:
IDK, I'd call this a bugfix.

It's a workflow fix for the issue where you tried to build the project and something mysterious went wrong but wasn't detected during the build.

But also as far as I can tell there's no 4.0 branch at all so I don't see why upstream would have an issue making this change at the bleeding edge of whatever development there *is*.

Maybe try it just to find out if it would be accepted? :)

Reply:
The active development in question is exactly one commit from November that is also cherry-picked to the 3.2 maintenance branch.

I'm familiar with repositories where actual, meaningful differences between branches exist. Even there, there's always a branch where changes *can* be applied... but in this case, it seems like it would actually be more work to have commits be exclusive to a particular branch.

Reply:
(Meson itself is an example of where one branch exists for active development while another branch exists to backport fixes.)

Reply:
Sure, like I said, I'll try to PR tests upstream when I have time. Until that (and a new release containing tests) happens, would it be possible to get this merged as-is, without tests?